### PR TITLE
add fr getting started page

### PR DESCRIPTION
**Pull request by apointeau**

### CODE REVIEW

**Comment by eed3si9n** [comment]
@apointeau Thanks for the contribution. I can't review this myself, but I created @scala/docs-fr team who might be able to. Maybe you could ping other French speakers you know of.

**Comment by eed3si9n** [comment]
I used to be able to add random ppl on the GitHub team, but I seem to have lost the authorization to do so. I'm going to mention a few ppl.

@Duhemm, @alexarchambault, @julienrf, @jto, @jeantil, @xbucchiotty, @Viewtiful Let us know if you could review French documentation translation. If you don't I'll find a Canadian person you would and you won't like it.

File: _fr/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+```
+- hello-world
+    - project (sbt utilise ce dossier pour ses propres fichiers)
+        - build.properties
+    - build.sbt (fichier de définition de la construction du package pour sbt)
+    - src
+        - main
+            - scala (tout votre code Scala doit être placé ici)
+                - Main.scala (Programme point d'entrée) <-- c'est tout ce dont nous avons pour le moment
```
Comment (jeantil) [inline _fr/getting-started/index.md:99]:
```suggestion
                - Main.scala (Programme point d'entrée) <-- c'est tout ce dont nous avons besoin pour le moment
```

File: _fr/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+Vous pouvez trouver plus de documentation à propose de sbt dans le [Scala Book](/scala3/book/tools-sbt.html) ([Lien](/overviews/scala-book/scala-build-tool-sbt.html) vers la version Scala 2) et sur la [documentation](https://www.scala-sbt.org/1.x/docs/index.html) officielle de sbt.
```
Comment (jeantil) [inline _fr/getting-started/index.md:102]:
```suggestion
Vous pouvez trouver plus de documentation à propos de sbt dans le [Scala Book](/scala3/book/tools-sbt.html) ([Lien](/overviews/scala-book/scala-build-tool-sbt.html) vers la version Scala 2) et sur la [documentation](https://www.scala-sbt.org/1.x/docs/index.html) officielle de sbt.
```

File: _fr/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec un IDE
+
+Vous pouvez ignorer le reste de cette page et aller directement sur [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html).
+
+
+## Ouvrir le projet hello-world
+
+Utilisons un IDE pour ouvrir le projet. Les plus populaires sont IntelliJ et VSCode.
+Il proposent tout deux de fonctionnalité riches, mais vous utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```
Comment (jeantil) [inline _fr/getting-started/index.md:112]:
```suggestion
Il proposent tout deux des fonctionnalités riches, mais vous utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```

File: _fr/getting-started/index.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec IntelliJ
+
+1. Téléchargez et installez [IntelliJ Community Edition](https://www.jetbrains.com/idea/download/)
+1. Installez l'extension Scala en suivant [les instruction IntelliJ pour installer des extensions](https://www.jetbrains.com/help/idea/managing-plugins.html)
+1. Ouvre le fichier `build.sbt` puis choisissez *Open as a project*
```
Comment (jeantil) [inline _fr/getting-started/index.md:118]:
```suggestion
1. Ouvrez le fichier `build.sbt` puis choisissez *Open as a project*
```

File: _fr/getting-started/index.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec VSCOde et metals
```
Comment (jeantil) [inline _fr/getting-started/index.md:120]:
```suggestion
### Avec VSCode et metals
```

File: _fr/getting-started/index.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+1. Téléchargez [VSCode](https://code.visualstudio.com/Download)
+1. Installez l'extension Metals depuis [la marketplace](https://marketplace.visualstudio.com/items?itemName=scalameta.metals)
+1. Ensuite, ouvrez le repertoire contenant le fichier `build.sbt` (cela doit être le dossier `hello-world` si vous avez suivi les insctructions précédentes). Choisissez *Import build* lorsque cela vous est demandé.
```
Comment (jeantil) [inline _fr/getting-started/index.md:124]:
```suggestion
1. Ensuite, ouvrez le repertoire contenant le fichier `build.sbt` (cela doit être le dossier `hello-world` si vous avez suivi les instructions précédentes). Choisissez *Import build* lorsque cela vous est demandé.
```

File: _fr/getting-started/index.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+
+> [Metals](https://scalameta.org/metals) est un "Serveur de langage Scala" qui fourni une aide pour écrire du code Scala dans VSCode et d'autres éditeurs [Atom, Sublime Text, autres ...](https://scalameta.org/metals/docs/editors/overview.html), en utilisant le protcole de langage serveur.
+> En arrière plan, Metals communique avec l'outil de construction de package en utilisant
+> le [Build Server Protocol (BSP)](https://build-server-protocol.github.io/).
+> Pour plus de détails sur le fonctionnement de Metals, suivez [“Write Scala in VS Code, Vim, Emacs, Atom and Sublime Text with Metals”](https://www.scala-lang.org/2019/04/16/metals.html).
+
+### Essayer avec le code source
```
Comment (jeantil) [inline _fr/getting-started/index.md:131]: Ce titre ne veut rien dire (this title makes no sense)

File: _fr/getting-started/index.md [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+
+Ouvrez ces deux fichiers dans votre IDE :
+
+- _build.sbt_
+- _src/main/scala/Main.scala_
+
+Quand vous allez lancer votre projet dans l'étape suivante, la configuration dans _build.sbt_ sera utilisé pour lance le code dans _src/main/scala/Main.scala_.
```
Comment (jeantil) [inline _fr/getting-started/index.md:138]:
```suggestion
Quand vous allez lancer votre projet dans l'étape suivante, la configuration dans _build.sbt_ sera utilisé pour lancer le code dans _src/main/scala/Main.scala_.
```

File: _fr/getting-started/index.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+## Lancer Hello Word
+
+Si vous êtes habitué à votre IDE, vous pouvez lancer le code dans _Main.scala_ depuis celui-ci.
+
+Sinon, vous pouvez lancer l'application depuis le terminal avec ces étapes :
+
+1. `cd` vers `hello-world`.
+1. Lancez `sbt`. Cela va ouvrir la console sbt.
+1. Ecrivez `~run`. Le symbole `~` est optionel, il va relancer l'application à chaque sauvegarde de fichier.
+   Cela permet un cyle rapide de modification/relance/debug. sbt va aussi génére un dossier `target` que vous pouvez ignorer.
```
Comment (jeantil) [inline _fr/getting-started/index.md:149]:
```suggestion
   Cela permet un cyle rapide de modification/relance/debug. sbt va aussi générer un dossier `target` que vous pouvez ignorer.
```

File: _fr/getting-started/index.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+
+Quand vous avec fini d'expérimenter avec ce projet, appuyez sur `[Entrée]` pour interrompre la commande `run`.
+Puis saisissez `exit` ou appuyer sur `[Ctrl+D]` pour quitter sbt et revenir à votre invite de commande.
```
Comment (jeantil) [inline _fr/getting-started/index.md:152]:
```suggestion
Puis saisissez `exit` ou appuyez sur `[Ctrl+D]` pour quitter sbt et revenir à votre invite de commande.
```

File: _fr/getting-started/index.md [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+
+## Prochaines étapes
+
+Une fois que vous avez terminé le tutoriel ce dessus, vous pouvre consulter :
```
Comment (jeantil) [inline _fr/getting-started/index.md:156]:
```suggestion
Une fois que vous avez terminé le tutoriel ce dessus, vous pouvez consulter :
```

File: _fr/getting-started/index.md [hunk 12]
```diff
@@ -0,0 +1,194 @@
...
+
+* [The Scala Book](/scala3/book/introduction.html) ([Lien](/overviews/scala-book/introduction.html) vers la version Scala 2), qui fourni un ensemble de courtes leçons et introduit les fonctionnalités principales de Scala.
+* [The Tour of Scala](/tour/tour-of-scala.html) pour une introduction des fonctionnalités Scala.
+* [Learning Resources](/learn.html), qui contient des tutoriels et des cours interactifs.
+* [Our list of some popular Scala books](/books.html).
+* [The migration guide](/scala3/guides/migration/compatibility-intro.html) pour vous aider à migrer votre code Scala 2 vers Scala 3.
+
+## Obtenir de l'aide
+Il y a plusieurs listes emails et cannaux de discussions instantanés si vous souhaitez rencontrer rapidement d'autres utilisateurs de Scala. Allez faire un tour sur notre page [community](https://scala-lang.org/community/) pour consulter la liste des ces ressources et obtenir de l'aide.
```
Comment (jeantil) [inline _fr/getting-started/index.md:165]:
```suggestion
Il y a plusieurs listes emails et canaux de discussions instantanés si vous souhaitez rencontrer rapidement d'autres utilisateurs de Scala. Allez faire un tour sur notre page [community](https://scala-lang.org/community/) pour consulter la liste des ces ressources et obtenir de l'aide.
```

File: _fr/getting-started/index.md (hunk 7 above)
Comment (jeantil) [reply _fr/getting-started/index.md:131]:
```suggestion
### Jouez avec le code source
```

**Comment by jeantil** [comment]
It reads mostly ok for me. I suggested a few changes to fix typos mostly.

**Comment by eed3si9n** [comment]
Un grand merci Jean! Antoine has opened like 8 PRs so maybe you and others could take a look at those too (it's late so later probably)? 🙏 https://github.com/scala/docs.scala-lang/pulls

**Comment by apointeau** [comment]
@jeantil Thank you very much, there was many small dumb mistakes, I will try to better reread the next times.

**Comment by jeantil** [comment]
@apointeau thanks for your work, it's always easier to spot these when you didn't write them :)

File: _fr/getting-started/index.md [hunk 13]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec un IDE
+
+Vous pouvez ignorer le reste de cette page et aller directement sur [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html).
+
+
+## Ouvrir le projet hello-world
+
+Utilisons un IDE pour ouvrir le projet. Les plus populaires sont IntelliJ et VSCode.
+Il proposent tout deux des fonctionnalités riches, mais vous utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```
Comment (mlachkar) [inline _fr/getting-started/index.md:112]:
```suggestion
Il proposent tout deux des fonctionnalités riches, mais vous pouvez utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```

File: _fr/getting-started/index.md [hunk 14]
```diff
@@ -0,0 +1,194 @@
...
+> Pour plus de détails sur le fonctionnement de Metals, suivez [“Write Scala in VS Code, Vim, Emacs, Atom and Sublime Text with Metals”](https://www.scala-lang.org/2019/04/16/metals.html).
+
+### Essayer avec le code source
+
+Ouvrez ces deux fichiers dans votre IDE :
+
+- _build.sbt_
+- _src/main/scala/Main.scala_
+
+Quand vous allez lancer votre projet dans l'étape suivante, la configuration dans _build.sbt_ sera utilisé pour lancer le code dans _src/main/scala/Main.scala_.
```
Comment (mlachkar) [inline _fr/getting-started/index.md:138]: Je me rends compte avec cette traduction que cette partie dans la page officiel n'apporte pas bcp à l'experience de démarrage en Scala, mais sinon c'est parfaitement traduit.

**Review by mlachkar — COMMENTED** [review]
La traduction est vraiment superbe! Merci beaucoup pour votre travail!

File: _fr/getting-started/index.md (hunk 14 above)
Comment (apointeau) [reply _fr/getting-started/index.md:138]: Merci beaucoup !

File: _fr/getting-started/index.md [hunk 15]
```diff
@@ -0,0 +1,194 @@
...
+Une fois que vous avez terminé le tutoriel ce dessus, vous pouvez consulter :
+
+* [The Scala Book](/scala3/book/introduction.html) ([Lien](/overviews/scala-book/introduction.html) vers la version Scala 2), qui fourni un ensemble de courtes leçons et introduit les fonctionnalités principales de Scala.
+* [The Tour of Scala](/tour/tour-of-scala.html) pour une introduction des fonctionnalités Scala.
+* [Learning Resources](/learn.html), qui contient des tutoriels et des cours interactifs.
+* [Our list of some popular Scala books](/books.html).
+* [The migration guide](/scala3/guides/migration/compatibility-intro.html) pour vous aider à migrer votre code Scala 2 vers Scala 3.
+
+## Obtenir de l'aide
+Il y a plusieurs listes emails et canaux de discussions instantanés si vous souhaitez rencontrer rapidement d'autres utilisateurs de Scala. Allez faire un tour sur notre page [community](https://scala-lang.org/community/) pour consulter la liste des ces ressources et obtenir de l'aide.
```
Comment (mlachkar) [inline _fr/getting-started/index.md:165]:
```suggestion
Il y a plusieurs listes de diffusion et canaux de discussions instantanés si vous souhaitez rencontrer rapidement d'autres utilisateurs de Scala. Allez faire un tour sur notre page [community](https://scala-lang.org/community/) pour consulter la liste des ces ressources et obtenir de l'aide.
```

**Review by mlachkar — APPROVED** [review]
Thanks again!

File: _fr/getting-started/index.md [hunk 16]
```diff
@@ -0,0 +1,194 @@
...
+### Utiliser la ligne de commande
+
+sbt est un outil de construction de package pour Scala, sbt compile, lance et teste votre code Scala.
+(Il peut aussi publier les librairies et faire beaucoup d'autres tâches.)
+
+Pour créer un nouveau projet Scala avec sbt :
+
+1. `cd` dans un répertoire vide.
+1. Lancez la commande `sbt new scala/scala3.g8` pour créer un projet Scala 3, ou `sbt new scala/hello-world.g8` pour créer un projet Scala 2.
+   Cela va télécharger un projet template depuis Github.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:86]:
```suggestion
   Cela va télécharger un projet modèle depuis Github.
```

File: _fr/getting-started/index.md [hunk 17]
```diff
@@ -0,0 +1,194 @@
...
+    <div id="download-step-one">
+        <p>Suivez <a href="https://get-coursier.io/docs/cli-installation.html#native-launcher" target="_blank">les instructions pour installer la commande <code>cs</code></a> puis exécutez :</p>
+        <p><code>$ ./cs setup</code></p>
+    </div>
+</div>
+
+En plus de gérer les JVMs, `cs setup` installe aussi des utilitaires en ligne de commande :
+
+- Un JDK (si vous n'en avez pas déjà un)
+- L'outil de construction de package [sbt](https://www.scala-sbt.org/)
```
Comment (Duhemm) [inline _fr/getting-started/index.md:40]: According to Wikipedia, the french translation of "build tool" is "moteur de production". I don't think this is necessarily clearer, but that avoids franglais. I'm sure @sjrd has a better suggestion?

File: _fr/getting-started/index.md [hunk 18]
```diff
@@ -0,0 +1,194 @@
...
+
+```
+- hello-world
+    - project (sbt utilise ce dossier pour ses propres fichiers)
+        - build.properties
+    - build.sbt (fichier de définition de la construction du package pour sbt)
+    - src
+        - main
+            - scala (tout votre code Scala doit être placé ici)
+                - Main.scala (Programme point d'entrée) <-- c'est tout ce dont nous avons besoin pour le moment
```
Comment (Duhemm) [inline _fr/getting-started/index.md:99]:
```suggestion
                - Main.scala (Point d'entrée du programme) <-- c'est tout ce dont nous avons besoin pour le moment
```

File: _fr/getting-started/index.md [hunk 19]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec un IDE
+
+Vous pouvez ignorer le reste de cette page et aller directement sur [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html).
+
+
+## Ouvrir le projet hello-world
+
+Utilisons un IDE pour ouvrir le projet. Les plus populaires sont IntelliJ et VSCode.
+Il proposent tout deux des fonctionnalités riches, mais vous pouvez utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```
Comment (Duhemm) [inline _fr/getting-started/index.md:112]:
```suggestion
Il proposent tout deux des fonctionnalités avancées, mais vous pouvez utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```

Comment (Duhemm) [inline _fr/getting-started/index.md:112]: I find the second part of the sentence "..., mais vous pouvez utiliser d'autres éditeurs." a bit weird, as it reads like a restriction or negation.

```suggestion
Il proposent tout deux des fonctionnalités riches. D'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html) sont également disponibles.
```

File: _fr/getting-started/index.md [hunk 20]
```diff
@@ -0,0 +1,194 @@
...
+
+### Avec IntelliJ
+
+1. Téléchargez et installez [IntelliJ Community Edition](https://www.jetbrains.com/idea/download/)
+1. Installez l'extension Scala en suivant [les instruction IntelliJ pour installer des extensions](https://www.jetbrains.com/help/idea/managing-plugins.html)
```
Comment (Duhemm) [inline _fr/getting-started/index.md:117]:
```suggestion
1. Installez l'extension Scala en suivant [les instruction d'IntelliJ pour installer des extensions](https://www.jetbrains.com/help/idea/managing-plugins.html)
```

File: _fr/getting-started/index.md [hunk 21]
```diff
@@ -0,0 +1,194 @@
...
+1. Ouvrez le fichier `build.sbt` puis choisissez *Open as a project*
+
+### Avec VSCode et metals
+
+1. Téléchargez [VSCode](https://code.visualstudio.com/Download)
+1. Installez l'extension Metals depuis [la marketplace](https://marketplace.visualstudio.com/items?itemName=scalameta.metals)
+1. Ensuite, ouvrez le repertoire contenant le fichier `build.sbt` (cela doit être le dossier `hello-world` si vous avez suivi les instructions précédentes). Choisissez *Import build* lorsque cela vous est demandé.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:124]:
```suggestion
1. Ensuite, ouvrez le `répertoire` contenant le fichier `build.sbt` (cela doit être le dossier `hello-world` si vous avez suivi les instructions précédentes). Choisissez *Import build* lorsque cela vous est demandé.
```

File: _fr/getting-started/index.md [hunk 22]
```diff
@@ -0,0 +1,194 @@
...
+
+> [Metals](https://scalameta.org/metals) est un "Serveur de langage Scala" qui fourni une aide pour écrire du code Scala dans VSCode et d'autres éditeurs [Atom, Sublime Text, autres ...](https://scalameta.org/metals/docs/editors/overview.html), en utilisant le protcole de langage serveur.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:126]:
```suggestion
> [Metals](https://scalameta.org/metals) est un "Serveur de langage Scala" qui fournit une aide pour écrire du code Scala dans VSCode et d'autres éditeurs [Atom, Sublime Text, autres ...](https://scalameta.org/metals/docs/editors/overview.html), en utilisant le [Language Server Protocol (LSP)](https://microsoft.github.io/language-server-protocol/).
```

File: _fr/getting-started/index.md [hunk 23]
```diff
@@ -0,0 +1,194 @@
...
+> Pour plus de détails sur le fonctionnement de Metals, suivez [“Write Scala in VS Code, Vim, Emacs, Atom and Sublime Text with Metals”](https://www.scala-lang.org/2019/04/16/metals.html).
+
+### Essayer avec le code source
+
+Ouvrez ces deux fichiers dans votre IDE :
+
+- _build.sbt_
+- _src/main/scala/Main.scala_
+
+Quand vous allez lancer votre projet dans l'étape suivante, la configuration dans _build.sbt_ sera utilisé pour lancer le code dans _src/main/scala/Main.scala_.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:138]:
```suggestion
Quand vous lancerez votre projet à l'étape suivante, la configuration dans _build.sbt_ sera utilisée pour lancer le code dans _src/main/scala/Main.scala_.
```

File: _fr/getting-started/index.md [hunk 24]
```diff
@@ -0,0 +1,194 @@
...
+
+## Lancer Hello Word
+
+Si vous êtes habitué à votre IDE, vous pouvez lancer le code dans _Main.scala_ depuis celui-ci.
+
+Sinon, vous pouvez lancer l'application depuis le terminal avec ces étapes :
+
+1. `cd` vers `hello-world`.
+1. Lancez `sbt`. Cela va ouvrir la console sbt.
+1. Ecrivez `~run`. Le symbole `~` est optionel, il va relancer l'application à chaque sauvegarde de fichier.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:148]:
```suggestion
1. Ecrivez `~run`. Le symbole `~` est optionnel, il va relancer l'application à chaque sauvegarde de fichier.
```

File: _fr/getting-started/index.md [hunk 25]
```diff
@@ -0,0 +1,194 @@
...
+   Cela permet un cyle rapide de modification/relance/debug. sbt va aussi générer un dossier `target` que vous pouvez ignorer.
+
+Quand vous avez fini d'expérimenter avec ce projet, appuyez sur `[Entrée]` pour interrompre la commande `run`.
+Puis saisissez `exit` ou appuyez sur `[Ctrl+D]` pour quitter sbt et revenir à votre invite de commande.
+
+## Prochaines étapes
+
+Une fois que vous avez terminé le tutoriel ce dessus, vous pouvez consulter :
+
+* [The Scala Book](/scala3/book/introduction.html) ([Lien](/overviews/scala-book/introduction.html) vers la version Scala 2), qui fourni un ensemble de courtes leçons et introduit les fonctionnalités principales de Scala.
```
Comment (Duhemm) [inline _fr/getting-started/index.md:158]:
```suggestion
* [The Scala Book](/scala3/book/introduction.html) ([Lien](/overviews/scala-book/introduction.html) vers la version Scala 2), qui fournit un ensemble de courtes leçons et introduit les fonctionnalités principales de Scala.
```

**Review by Duhemm — COMMENTED** [review]
I've found a few typos and made a few suggestions. Thanks @apointeau !

**Comment by mlachkar** [comment]
@Duhemm I will add your suggestions to a new PR!

**Pull request merged**